### ISSUE TITLE
Mac OS 12

### ISSUE THREAD

**Issue by edludlowedit** [jwise/HoRNDIS#156]
Anyone had any luck getting this going Mac OS 12 Monterey?

**Comment by Jasongamerkid14** [comment]
i do. im in big sur

**Comment by Barathwaja** [comment]
Yes it's not working for me as well. I'm using Mac OS 12.4 Monterey (Intel Chip)

**Comment by ea7kir** [comment]
Will there be any further development or is this project dead?

**Comment by tfvglin** [comment]
is there anyone working on this project?

**Comment by MicahZheng** [comment]
https://youtu.be/32lM27TGNFM
this video worked well for me ,I'm using Mac OS 12.4 Monterey (Intel Chip)

First you need to disable SIP in recovery mode using "csrutil disable"

[HoRNDIS.kext.zip](https://github.com/jwise/HoRNDIS/files/9137322/HoRNDIS.kext.zip)

@ea7kir @tfvglin @Barathwaja @

**Comment by ghost** [comment]
Any alternative to this which doesn't require disabling SIP?

**Comment by egigoka** [comment]
> https://youtu.be/32lM27TGNFM this video worked well for me ,I'm using Mac OS 12.4 Monterey (Intel Chip)
> 
> First you need to disable SIP in recovery mode using "csrutil disable"
> 
> [HoRNDIS.kext.zip](https://github.com/jwise/HoRNDIS/files/9137322/HoRNDIS.kext.zip)
> 
> @ea7kir @tfvglin @Barathwaja @

csrutil enable --kext 
is enough btw

**Comment by LizGalanis** [comment]
I'm trying to install this on Monterey 12.5.1 M1 chip with SIP disabled but I get an error saying it needs to be updated by the developer. Anyone managed to use this recently?

**Comment by egigoka** [comment]
> I'm trying to install this on Monterey 12.5.1 M1 chip with SIP disabled but I get an error saying it needs to be updated by the developer. Anyone managed to use this recently? 

Did you tried builds specially for arm? https://github.com/nEdAy/HoRNDIS-for-M1 like this one?

**Comment by timredfern** [comment]
On OSX Monterey 12.5.1 (intel chip), I disabled SIP with csrutil and used the download package to install HoRNDIS 9.2. It seems to work well, impressive performance.

**Comment by timredfern** [comment]
I'm finding that the connection goes stale after a short time and I need to restart the driver. I think this is a problem with HoRNDIS on OSX12 rather than with my phone, as I don't see the problem on a Macbook running Mojave.

**Comment by LizGalanis** [comment]
> https://github.com/nEdAy/HoRNDIS-for-M1

Thanks, I have tried this one but still not working. My computer now says it is successful when I install the app, however trying to tether my phone does not work. I have discovered that in the system information it has listed the Horndis app under 'disabled software' for no user consent. I don't know how to consent! Can anyone help with this?

**Comment by LizGalanis** [comment]
Is anyone able (and willing) to get on a call with me to help me make this work?

**Comment by egigoka** [comment]
> Is anyone able (and willing) to get on a call with me to help me make this work? 

https://t.me/filthymold
Contact me, I'll try to help

**Comment by philipmoulton** [comment]
> > https://youtu.be/32lM27TGNFM this video worked well for me ,I'm using Mac OS 12.4 Monterey (Intel Chip)
> > First you need to disable SIP in recovery mode using "csrutil disable"
> > [HoRNDIS.kext.zip](https://github.com/jwise/HoRNDIS/files/9137322/HoRNDIS.kext.zip)
> > @ea7kir @tfvglin @Barathwaja @
> 
> csrutil enable --kext is enough btw

This spits out an error about `--kext` being an invalid argument. 
I'm guessing what you intended was(or I'm on a different version of the `csrutil` tool): 
`csrutil enable --without kext`
Running this in the Recovery Mode terminal allowed me to run `sudo kextload /Library/Extensions/HoRNDIS.kext` successfully and enable the extension from Security & Privacy settings.
